pallets/proof_of_existence/src/weight.rs

```diff
@@ -22,7 +22,7 @@
 //! WASM-EXECUTION: `Compiled`, CHAIN: `Some("dev")`, DB CACHE: `1024`
 
 // Executed Command:
-// /home/mdamico/devel/NH-core/target/production/nh-node
```

Comment:
This can be opinionated, so evaluate if actually to be considered.
Regardless of file autogeneration (and probably being this overwritten in the next future), since all weigths file has been modified within the PR I would also adapt:

- `the New Horizen node` to `the zkVerify node`,
- `NH-core/node` to `zkVerify/node`,
- `bin/nh-node` to `bin/zkv-node`.

This comment applies to all weight files.

Reply:
I think that:
- The first comment is addressed by the change in `node/hl-deploy-weight-template.hbs`
- The second one is due to the fact that whoever run the benchmark still had the old "NH-core" name (didn't re-clone the repo when we changed the name)
- Same goes for the third one